conda_forge_webservices/update_me.py

```diff
@@ -102,22 +107,27 @@ def main():
                 fp.write("\n")
             repo.index.add(pth)
 
-            msg_vers = (
-                "Redeploy for package updates:\n\n" +
-                "\n".join(["* `{}={}`".format(k, v) for k, v in to_install.items()])
-            )
+            if len(to_install) > 1:
+                msg = (
+                    "Redeploy for package updates\n\n" +
+                    "\n".join(["* `{}={}`".format(k, v) for k, v in to_install.items()])
+                )
+            else:
+                (k, v), = to_install.items()
+                msg = f"Redeploy for package update: {k}={v}"
```

Comment:
Maybe we could use code format for the package version

No strong feelings though

```suggestion
                msg = f"Redeploy for package update: `{k}={v}`"
```